openprovider/data/exception_map.py

```diff
@@ -265,6 +298,15 @@
     872: NoSuchElement,             # Zone specified is not found.
     873: NoSuchElement,             # Template specified is not found.
     874: ValidationError,           # Template ID is empty.
+    875: RuleViolation,             # Not enough nameservers given
+    876: RuleViolation,             # It is not possible to use MX records that point to an IP address in the range 79.99.129.0/24.
+    890: RuleViolation,             # Incorrect DNSSEC record format.
+    891: RuleViolation,             # DNSSEC is not supported for this extension
+    892: RuleViolation,             # Protocol field of the dnskey record must be 3
+    893: RuleViolation,             # Algorithm field has wrong value
+    894: RuleViolation,             # Digest type field has wrong value
+    895: RuleViolation,             # DNSSEC keys and digests should be BASE64 encoded
+    896: RuleViolation,             # DNSSEC processing error
```

Comment:
I had that problem with a lot of these errors. Hard to find out just by the error message. Downgrade to the generic exception?